deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -46,6 +47,11 @@ class TrainTestPredictionDrift(TrainTestCheck, ReduceMixin):
 
     Parameters
     ----------
+    drift_mode: str, default: 'auto'
+        Controls whether to compute drift on the predicted probabilities or the predicted classes in case of a
+        classification task. If 'auto', compute drift on the predicted probabilities if the task is multiclass, and on
+        the predicted classes otherwise.Set to 'proba' to force drift on the predicted probabilities, and 'prediction'
```

Comment:
```suggestion
        the predicted classes otherwise. Set to 'proba' to force drift on the predicted probabilities, and 'prediction'
```

Reply:
copy cat.

Reply:
Same here, it this the default behaviour we agreed on?

Reply:
+ the description for drift_mode is a bit confusing if your task is regression